### DISCUSSION TITLE
Hubs Cloud Support for Asia Pacific(Jakarta) Region.

### DISCUSSION THREAD

**Discussion by rahmatullah5** [mozilla/hubs#5059] [Hubs Cloud Questions]
Hi, I'd like to ask a question regarding the hubs  cloud https://aws.amazon.com/marketplace/pp/prodview-2kj77cx5ya7du#pdp-reviews

Is there any plan to support the Asia Pacific (Jakarta) ap-southeast-3 region?

Thanks in advance

#hubs-cloud

**Comment by matthewbcool** [comment]
Looks like this question has been answered and we can follow the thread here: https://github.com/mozilla/hubs/discussions/5058
